client/src/components/ProfileCertifications/ProfileCertifications.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
```

Comment:
Same as the past PR, no need to have the React import in Vite.

Reply:
I do need this import because I'm using <React.Fragment> component

Reply:
Oh! That's right, I forgot about that component, my bad.